jetson/teleop/src/__main__.py

```diff
@@ -257,15 +248,15 @@ def gimbal_control_callback(channel, msg):
     yawData = [keyboardData.a - keyboardData.d,
                keyboardData.j - keyboardData.l]
 
-    gimbal_msg = GimbalCmd()
+    gimbal_msg = MastGimbalCmd()
     gimbal_msg.pitch = pitchData
     gimbal_msg.yaw = yawData
 
-    lcm_.publish('/gimbal_openloop_cmd', gimbal_msg.encode())
+    lcm_.publish('/mast_gimbal_cmd', gimbal_msg.encode())
 
 
 def main():
-    hb = heartbeatlib.OnboardHeartbeater(connection_state_changed, 0)
+    hb = heartbeatlib.JetsonHeartbeater(connection_state_changed, 0)

```

Comment:
nit JetsonHeartBeater

Reply:
It's the name of the class in rover_commons/src if we want to change it there?

Reply:
Seems to be pretty far reaching, I got some errors I couldn't figure out immediately